### PR TITLE
KVP-BASIC-CHECKS - Modify the result of the KVP test to skip when daemon is not running by default

### LINKED ISSUE #371: KVP test should be skipped if daemon is not running by default
https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/KVP-BasicChecks.sh#L44

### PR DESCRIPTION
**Pull request by Tong-Xu**
Fix https://github.com/LIS/LISAv2/issues/371

After fix-
```
[LISAv2 Test Results Summary]
Test Run On           : 06/27/2019 02:05:53
ARM Image Under Test  : RedHat : RHEL : 7-RAW : 7.6.2019062120
Total Test Cases      : 1 (0 Passed, 0 Failed, 0 Aborted, 1 Skipped)
Total Time (dd:hh:mm) : 0:0:6

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 KVP                  KVP-BASIC-CHECKS                                                               SKIPPED                 2.74 

[LISAv2 Test Results Summary]
Test Run On           : 06/27/2019 02:22:57
ARM Image Under Test  : CoreOS : CoreOS : Stable : 2135.4.0
Total Test Cases      : 1 (0 Passed, 0 Failed, 0 Aborted, 1 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 KVP                  KVP-BASIC-CHECKS                                                               SKIPPED                 2.69 
```

### CODE REVIEW

File: Testscripts/Linux/KVP-BasicChecks.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ fi
 
 if [ "${kvp_items:-UNDEFINED}" = "UNDEFINED" ]; then
     LogErr "The test parameter kvp_items is not defined in constants.sh"
-    SetTestStateAborted
+    SetTestStateSkipped
```
Comment (chvalean) [inline Testscripts/Linux/KVP-BasicChecks.sh]: This approach is not very good.

Lines 23-36 must be moved after the check if KVP daemon is running (and don't skip there).

Current line 44 that does the kvp daemon check must exit with Skip if it doesn't find it running.

**Review by chvalean — CHANGES_REQUESTED** [review]

File: Testscripts/Linux/KVP-BasicChecks.sh (hunk 1 above)
Comment (LiliDeng) [reply Testscripts/Linux/KVP-BasicChecks.sh]: @chvalean thanks, Lines 23-36 do the check values from constants.sh file, put it before or after check kvp, does it matter?

Comment (LiliDeng) [reply Testscripts/Linux/KVP-BasicChecks.sh]: Ah, understood.

File: Testscripts/Linux/KVP-BasicChecks.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ fi
 pid=$(pgrep "hypervkvpd|hv_kvp_daemon")
 if [ $? -ne 0 ]; then
     LogErr "KVP Daemon is not running by default"
```
Comment (chvalean) [inline Testscripts/Linux/KVP-BasicChecks.sh]: please change this to LogMsg.
Also do the same message with UpdateSummary, so that we have it handy.

**Review by chvalean — APPROVED** [review]

**Pull request merged**